classes/class08/README.md

```diff
@@ -1 +1,101 @@
+
+Time to write some tests then. In `test.js`, uncomment up through line 5, so you have:
+```// Setup our assertion library
+var expect = require('chai').expect;
+
+var index = require('../../routes/index');```
+The first line loads the [Chai](http://chaijs.com/) assertion library and assigns it's 'expect' test syntax to the variable `expect`, which we will use to write tests. The second line just loads our index route, because we will want to test that. If we run our tests again, we should still see nothing passing, but no errors. Now uncomment up to line 17, as well as line 22, leaving you with:
+```
+describe("A test suite", function() {
+	// Syncronous
+	it('should pass', function() { 
+		expect(true).to.be.true; 
+	});
+});
+```
+Now running the tests should tell us that 1 test is passing! Let's take a look at what we just did. First off, we call the `describe` function, which creates our test suite. It takes a string, which servers as the name of the test suite, as well as a function with no arguments that runs the tests in the suite. Next we call `it`, which sets up a test for some piece of functionality, taking first the name of that test, and then a function, in this case with no arguments. Inside that function we finally make an assertion using `expect`, which we loaded from chai earlier. `expect` takes some value and returns an object with a bunch of useful comparison methods that will compare to the object we passed in. In this case, `expect(true).to.be.true` means, in English, "Take the value 'true' and see if it is truthy" which, of course, it is. The syntax here probably feels a little weird, and in fact t is. the `to` and `be` methods actually do nothing, they just chain together because somebody, sometime decided it was a good idea to be able to write test assertions that sound like English. We happen to agree with them, but if you do not, check out Chai's other assertion syntaxes and find one you like, they all have the same functionality. `true` is what actually does a comparison. All you really need to know is that it checks what comes before it to see if it is truthy, and if so it passes the test and otherwise it fails, and mocha takes care of the details of reporting that out. Every asserion you write will start with a call to `expect`, have some chaining methods (or maybe none), and some comparison method at the end. Chai's assertion methods are documented [here](http://chaijs.com/api/bdd/).
```

Comment:
I'd argue that it might be just as productive to use this as an example of "semantic" method naming and structuring. The line of test code pretty much reads like English and it's not totally useless since you can do stuff like 

```node
expect(x).not.to.be.true
```